capabilities/docker/docker.go

```diff
@@ -0,0 +1,50 @@
+package docker
```

Comment:
I'd be inclined to use a less generic package name/id for this capability, not a blocker tho.